knife/lib/chef/knife/bootstrap.rb

```diff
@@ -67,6 +67,10 @@ class Bootstrap < Knife
         description: "The number of seconds to wait for each connection operation to be acknowledged while running bootstrap.",
         default: 60
 
+      option :request_tty,
+        long: "--request-tty",
+        description: "Setting the non_interactive to false when request-tty is true",
```

Comment:
can update description as - `Enable interactive SSH terminal to allow for password prompts.`
can set default - `boolean: false`